docs/pages/guides/testing-with-jest.mdx

```diff
@@ -182,3 +183,9 @@ This is optional, but wanted to talk about different Jest test flows. Currently
   "updateSnapshots": "jest -u --coverage=false"
 }
 ```
+
+For more information, see [CLI Options](https://jestjs.io/docs/en/cli).
```

Comment:
```suggestion
For more information, see [CLI Options](https://jestjs.io/docs/en/cli) in Jest documentation.
```

Not sure if the form is correct, but I think it's always nice to inform user that he will leave our docs.